### DISCUSSION TITLE
webdav无法通过网页进入，windows磁盘映射也没用。

### DISCUSSION THREAD

**Discussion by coco-w** [AlistGo/alist#1339] [Q&A]
网页浏览是可以的
![image](https://user-images.githubusercontent.com/42274898/179016141-5e1beb79-cfe7-4e13-b4e4-0adc4f995be8.png)
但是使用dav的网站就会自动跳到
![image](https://user-images.githubusercontent.com/42274898/179016368-045cc3c6-48c0-43ba-b525-691856fef5da.png)
磁盘映射也打不开。
但是在kodi中添加webdav是可以使用的
![image](https://user-images.githubusercontent.com/42274898/179016681-5668fee5-f10d-4209-9267-3efcab941d8d.png)

**Comment by rakaki** [comment]
使用raidrive映射为磁盘，webdav兼容性比较差，手机端可以用nplayer

**Reply by coco-w** [reply]
已经解决，windows开启支持http webdav访问

**Reply by xinxinenjoy** [reply]
> 已经解决，windows开启支持http webdav访问

能请教一下您这个是怎么操作的么？  面临一样的问题，KODI无法用WEBDAV访问，先挂载本地磁盘再分享太麻烦了
